lib/kanta/migrations/sqlite3/v02.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Kanta.Migrations.SQLite3.V02 do
+  @moduledoc """
+  Kanta V2 Migrations
+  """
+
+  use Ecto.Migration
+
+  @kanta_application_sources "kanta_application_sources"
+  @kanta_messages "kanta_messages"
+
+  def up(opts) do
+    [
+      &up_application_sources/1,
+      &up_kanta_messages/1
+    ]
+    |> Enum.each(&apply(&1, [opts]))
+  end
+
+  def down(opts) do
+    [
+      &down_application_sources/1,
+      &down_kanta_messages/1
+    ]
+    |> Enum.each(&apply(&1, [opts]))
+  end
+
+  def up_application_sources(_opts) do
+    create_if_not_exists table(@kanta_application_sources) do
+      add(:name, :string)
+      add(:description, :text)
+      add(:color, :string, null: false, default: "#7E37D8")
```

Comment:
If this is the same color as before, maybe even better option is to move it to some config / utils module?